.travis.yml

```diff
@@ -91,16 +90,15 @@ matrix:
         - git remote add gkwillie https://gkwillie:${GKWILLIE_TOKEN}@github.com/$TRAVIS_REPO_SLUG
         - git push --tags gkwillie HEAD:master
         - set +e
-    - if: branch IN (master, production) AND type != pull_request AND commit_message =~ /^Bump version/
+    - if: branch = master AND type != pull_request AND commit_message =~ /^Bump version/
```

Comment:
So there is no more `production` branch, correct?

Reply:
from [[ENG-275]] we will have pinned services versions on prod, so since that issue is done we will deploy on prod only the images that are mentioned in specific config files, that means that trying to update the service on production from production branch is still possible but not recommended.